Makefile

```diff
@@ -49,3 +51,26 @@ tester-image: ## Build docker image for the tester component
 kind-load: image tester-image ## Load the required image to KinD cluster
 	kind load docker-image $(REPO):latest
 	kind load docker-image $(TESTER_REPO):v1
+
+.PHONY: images
+images: ## Build multi arch docker images for the project
+	docker buildx build \
+		--platform $(SUPPORTED_ARCHS) \
+		--output "type=image,push=false" \
```

Comment:
AIUI, since we are building multi-platform images (using builders with `docker-container` driver), this doesn't output any images, right? (See ee9b892c82f6.) So is the purpose of this rule simply to validate the image build? Do we need this target? Or shall we just provide a single target that can build and push the multi-platform images, and we **don't** default `REGISTRY` to `gcr.io/linkerd-io`?

Reply:
yes, this is not output any image, meaning the images will not appear when we call `docker images` but the images will exist in the cache layer.

Yes, validation is the purpose and also clarity between build & push (on CI stages). The push part will just use the previous cache from the build.

Reply:
For the default registry, for now, we use `gcr.io/linkerd-io` but it can be easily changed using env variable
`DOCKER_REGISTRY=<registry> make ....`

Reply:
Thanks for the clarification. Is it important to have 2 distinct phases in CI, since the build phase is essentially the same as the push, minus the actual push task? 

Just wondering if can we do something like:

```
images:
	docker buildx build \
		--platform $(SUPPORTED_ARCHS) \
		--output "type=image,push=${PUSH_IMAGE}" \
# ...
```
setting `PUSH_IMAGE` to `true` in CI, but `false` by default, and remove the `push` target?

Reply:
important maybe not really, but just for distinction. 

that is a good suggestion, I will change it, to make it DRY.
Regarding removing the `push`, we can still use it, so it's just calling `PUSH_IMAGE=true  make images`. So the caller no needs to know which env variable to set.